src/ui/components/DefinitionList/index.js

```diff
@@ -0,0 +1,50 @@
+/* @flow */
+import * as React from 'react';
+import makeClassName from 'classnames';
+
+import './styles.scss';
+
+
+type DefinitionItemProps = {|
+  children: React.Node,
+  className?: string,
+  title: React.Node,
+|};
+
+export const DefinitionItem = (
+  { children, className = '', title }: DefinitionItemProps = {}
+) => {
+  return (
+    <React.Fragment>
+      <dt
+        className={makeClassName(className, {
+          [`${className}-title`]: className && className.length,
+        })}
+      >
+        {title}
+      </dt>
+      <dd
+        className={makeClassName(className, {
+          [`${className}-contents`]: className && className.length,
+        })}
+      >
+        {children}
+      </dd>
+    </React.Fragment>
+  );
+};
+
+type DefinitionListProps = {|
+  className?: string,
+  children: React.Node,
```

Comment:
You may get some helpful Flow validation if you restrict this to the only type of child the component accepts:

```js
children: React.ChildrenArray<React.Element<typeof DefinitionList>>,
```

Reply:
Oh, I'll do that.